docs/en/operations/table_engines/join.md

```diff
@@ -1,36 +1,52 @@
-
 # Join
 
-A prepared data structure for JOIN that is always located in RAM.
+Prepared data structure for using in [JOIN](../../query_language/select.md#select-join) operations.
+
+## Creating a table
 
 ```
-Join(ANY|ALL, LEFT|INNER, k1[, k2, ...])
+CREATE TABLE [IF NOT EXISTS] [db.]table_name [ON CLUSTER cluster]
+(
+    name1 [type1] [DEFAULT|MATERIALIZED|ALIAS expr1] [TTL expr1],
+    name2 [type2] [DEFAULT|MATERIALIZED|ALIAS expr2] [TTL expr2],
+) ENGINE = Join(join_strictness, join_type, k1[, k2, ...])
 ```
 
-Engine parameters: `ANY|ALL` – strictness; `LEFT|INNER` – type. For more information, see the [JOIN Clause](../../query_language/select.md#select-join) section.
-These parameters are set without quotes and must match the JOIN that the table will be used for. k1, k2, ... are the key columns from the USING clause that the join will be made on.
+See the detailed description of [CREATE TABLE](../../query_language/create.md#create-table-query) query.
 
-The table can't be used for GLOBAL JOINs.
+**Engine Parameters**
 
-You can use INSERT to add data to the table, similar to the Set engine. For ANY, data for duplicated keys will be ignored. For ALL, it will be counted.
+- `join_strictness` – `JOIN` strictness.
+- `join_type` – `JOIN` type.
+- `k1[, k2, ...]` – Key columns from the `USING` clause that the `JOIN` operation is made with.
 
-You can't perform SELECT directly from the table. There are two ways to retrieve data from table with Join engine
-* use it as the "right-hand" table for JOIN
-* use `joinGet` function, which allows to extract data from Join table with dictionary-like syntax.
+Set the parameters `join_strictness` and `join_type` without quotes, for example, `Join(ANY, LEFT, col1)`. They must match the `JOIN` operation that the table will be used for. For more information about `JOIN` type and strictness, see the `JOIN` clause description.
```

Comment:
probably should mention what happens when they mismatch (exception or incorrect result?)